data/sql/updates/pending_db_world/gostly-philantropist-to-jail.sql

```diff
@@ -0,0 +1,3 @@
+--
+UPDATE `creature_template` SET `mingold` = 0, `maxgold` = 0
+WHERE `entry` = 16470;
```

Comment:
You can keep it in one single line
```suggestion
--
UPDATE `creature_template` SET `mingold` = 0, `maxgold` = 0 WHERE `entry` = 16470;
```